prime-angular-frontend/src/app/modules/adjudication/shared/components/enrollee-table/enrollee-table.component.ts

```diff
@@ -60,6 +60,10 @@ export class EnrolleeTableComponent implements OnInit, OnChanges {
   public EnrolmentStatus = EnrolmentStatusEnum;
   public Role = Role;
   public readonly PAPER_ENROLLEE_GPID_PREFIX = PAPER_ENROLLEE_GPID_PREFIX;
+  public readonly paperEnrolmentStatusMap = new Map<number, string>([
```

Comment:
can make this a map between EnrolmentStatusEnum and string, then remove the comments on the following lines.